packages/scandipwa/src/component/AddIcon/AddIcon.style.scss

```diff
@@ -12,7 +12,12 @@
 .AddIcon {
     @include icon-black-hover-effect;
 
-    cursor: pointer;
+    &:not([disabled]) {
+        cursor: pointer;
+    }
+    &:disabled {
+        cursor: default;
+    }
```

Comment:
Please, explain why 
```
&:disabled {
        cursor: default;
    }
```
is required? Will the cursor be not the default arrow, if not explicitly overridden to  `cursor: default;`?

Reply:
yes, without specifying that when it is disabled it should be cursor: default it would stay "hand" instead of normal cursor